ConsentViewController/Classes/Consents/SPUserData.swift

```diff
@@ -54,6 +54,25 @@ extension SPConsent {
     }
 
     public override var description: String { "gdpr: \(String(describing: gdpr)), ccpa: \(String(describing: ccpa))" }
+
+    open override func isEqual(_ object: Any?) -> Bool {
```

Comment:
Does the parameter `object` need to be of type `Any?`? Or could you change it to `SPUserData` and remove that casting (`as?`) from the code?

Reply:
@andresilveirah ` override func isEqual(_ object: SPUserData) ` goes `Method does not override any method from its superclass`
since `NSObject`'s method declaration is ` override func isEqual(_ object: Any?) `
I can: 
1) get rid of `override` by using overloading ( ` func isEqual(_ object: SPUserData) ` ) 
or 
2) override the ` func ==(left: SPUserData, right: SPUserData) -> Bool `

Reply:
Oh, my bad, just saw the standard implementation of all `func isEqual` takes `Any?` as param. Let's stick with it.